api/src/main/java/uk/gov/dhsc/htbhf/claimant/entity/Address.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.dhsc.htbhf.claimant.entity;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+/**
+ * Domain object for an Address.
+ */
+@Entity
+@Data
+@Builder
+@Table(name = "address")
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor
+public class Address {
+
+    @Id
+    @Getter(AccessLevel.NONE)
+    @Access(AccessType.PROPERTY)
+    private UUID id;
+
+    @OneToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "id")
+    private Claimant claimant;
+
+    @NotNull
+    @Size(min = 1, max = 500)
+    @Column(name = "address_line_1")
+    private String addressLine1;
+
+    @Size(max = 500)
+    @Column(name = "address_line_2")
+    private String addressLine2;
+
+    @NotNull
+    @Size(min = 1, max = 500)
+    @Column(name = "townOrCity")
```

Comment:
Column name doesn't match the format of other column names (underscores)

Reply:
In fact the column name doesn't match, we should have a failing unit test for this...

Reply:
I've updated the name but uncertain as to why the repository test passed. Potentially some hibernate magic or h2 mismatch? Once I've added the api layer in I'll see if this breaks the application.